.github/workflows/main.yaml

```diff
@@ -10,66 +10,69 @@ on:
     paths-ignore: ["**.md"]
 
 jobs:
-  lint:
+  lint-and-test:
     if: github.event_name == 'pull_request'
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Validate
+      - name: Setup Go environment
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15
+      - name: Setup dependencies
+        run: make setup
+      - name: Lint tasks
         run: |
           make validate-versions
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.37.1
```

Comment:
if we change the golangci-lint version we need to update bin/setup.sh https://github.com/fidelity/kraan/blob/e23e382980e25408e5f9412fba1e63d4858dd885/bin/setup.sh#L3 and docs/dev-guide.md too